x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/report_metric_options.tsx

```diff
@@ -73,24 +79,43 @@ export function ReportMetricOptions({ seriesId, series, seriesConfig }: Props) {
   });
 
   return (
-    <EuiSuperSelect
-      fullWidth
-      options={
-        series.selectedMetricField
-          ? options
-          : [
-              {
-                value: SELECT_REPORT_METRIC,
-                inputDisplay: SELECT_REPORT_METRIC_LABEL,
-                disabled: false,
-              },
-              ...options,
-            ]
-      }
-      valueOfSelected={series.selectedMetricField || SELECT_REPORT_METRIC}
-      onChange={(value) => onChange(value)}
-      style={{ minWidth: 220 }}
-    />
+    <>
+      {!series.selectedMetricField && (
+        <EuiPopover
+          button={
+            <EuiButton
+              iconType="plusInCircle"
+              onClick={() => setShowOptions((prevState) => !prevState)}
+              fill
+              size="s"
+            >
+              {SELECT_REPORT_METRIC_LABEL}
+            </EuiButton>
+          }
+          isOpen={showOptions}
+          closePopover={() => setShowOptions((prevState) => !prevState)}
+        >
+          <EuiListGroup>
+            {options.map((option) => (
+              <EuiListGroupItem
+                key={option.value}
+                onClick={() => onChange(option.value)}
+                label={option.dropdownDisplay}
+                isDisabled={option.disabled}
+              />
+            ))}
+          </EuiListGroup>
+        </EuiPopover>
+      )}
+      {series.selectedMetricField && (
+        <EuiBadge>
+          {
+            seriesConfig?.metricOptions?.find((option) => option.id === series.selectedMetricField)
+              ?.label
+          }
+        </EuiBadge>
+      )}
+    </>
```

Comment:
In edit/expanded mode, we should allow user to change the metric option. Since report metric can be editable.

Reply:
@liciavale How do you see this interaction happening? What does the edit state look like for the report metric? Does it revert back to a button?

Reply:
In my design, it is a badge that dismissed become a + add button again and we show the message that indicates that metric is missing. 
After a chat with Shahzad we decided to try to use a single select without creating new behavior and my assumption is that it will always be there after they add a metric but I don't have that in my design tho' so I'm not sure how this will look like. Would you like me to change the design?